packages/apidom-ls/src/services/validation/linter-functions.ts

```diff
@@ -1009,9 +1009,37 @@ export const standardLinterfunctions: FunctionItem[] = [
     functionName: 'apilintOpenAPIPathTemplateValid',
     function: (element: Element) => {
       if (isStringElement(element)) {
-        return true;
+        const parseResult = parse(toValue(element));
+        const parts: string[] = [];
+        parseResult.ast.translate(parts);
+
+        const templateExpressions = parts
```

Comment:
We can safely drop lines 1016-1022. This is already guaranteed by condition to this rule: https://github.com/swagger-api/apidom/blob/b0b03a2d2549a68701248909d7b73ec5a1d51ae5/packages/apidom-ls/src/config/openapi/path-template/lint/value--valid.ts#L15

Reply:
You mean this part?
```typescript
        if (templateExpressions.length === 0) {
          return true;
        }
```
I still need the template expressions list so this:
```typescript
const templateExpressions = parts
          .filter((part) => part[0] === 'template-expression')
          .map((part) => part[1].slice(1, -1));
``` 
has to stay

Reply:
You're right. We can drop just part of the code.

Reply:
Sure, removed.